tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2410,6 +2410,41 @@ Status ConvertTranspose(OpConverterParams* params) {
   return Status::OK();
 }
 
+Status ConvertShape(OpConverterParams* params) {
+  const auto& inputs = params->inputs;
+  const auto& node_def = params->node_def;
```

Comment:
Can we remove this variable? It is not used when IS_TRT_VERSION_GE(6, 0, 0, 0) is not true.

Reply:
Removed.